src/query/storages/hive/src/hive_table.rs

```diff
@@ -240,13 +240,34 @@ impl HiveTable {
             .get_partition_names(table_info[0].to_string(), table_info[1].to_string(), -1)
             .await?;
 
+        if tracing::enabled!(tracing::Level::TRACE) {
```

Comment:
Can we use the `tracing::instrument` span instead of the log? https://databend.rs/doc/faq/how-to-tracing#distributed-tracing-with-jaeger

cc @drmingdrmer

Reply:
could you explain the reason why using `tracing::instrument` ?

Reply:
> Can we use the `tracing::instrument` span instead of the log? https://databend.rs/doc/faq/how-to-tracing#distributed-tracing-with-jaeger
> 
> cc @drmingdrmer

Yes. Then there will be a span that includes all the events logged under this function.
But this if-block will print a very long log that may not fit in a tracing span, IMHO.